lib/hlclock/server.ex

```diff
@@ -9,6 +9,7 @@ defmodule HLClock.Server do
   end
 
   def init(opts) do
+    Process.send_after(self(), :periodic_send, interval())
```

Comment:
I guess there's no need for us to store the timer reference.

Reply:
I don't see how we could use it effectively without greatly increasing the complexity of the send and recv calls

Reply:
Yeah thats fine.